lib/entities/movie.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+
+class Movie {
+  final String title;
+  final String originalTitle;
+  final genres = const [
+    'Animation',
+    'Family',
+    'Adventure',
+    'Fantasy',
+    'Comedy'
+  ];
```

Comment:
I'd leave the genres as an argument as well, as those genres are just for your mocked movie!